src/App.js

```diff
@@ -73,22 +82,26 @@ querySearch = (query = 'tech') => {
         </div>
       </div>
       <Search onSearch={this.querySearch}/> 
-      <div className='main-nav'>
-        <Nav /*onClick={this.querySearch}*//>
       </div>  
+      <div className='main-nav'>
+          <Nav /> 
+      </div>
       <div className="main-content">
           <Switch> 
-              <Route exact path="/" render={() => <PhotoContainer data={this.state.mountains} />}/>
-              <Route path="/trees" render={() => <PhotoContainer data={this.state.trees} />}/>
-              <Route path="/stars" render={() => <PhotoContainer data={this.state.stars} />}/>
-              <Route path={"/search/:query"} render= {() => <PhotoContainer data={this.state.photos} />}/>
+              <Route exact path ="/" render={() => <Redirect to="/search/mountains" />} /> 
+              <Route path="/search/mountains" render={() => <PhotoContainer data={this.state.mountains} />}/>
+              <Route path="/search/trees" render={() => <PhotoContainer data={this.state.trees} />}/>
+              <Route path="/search/stars" render={() => <PhotoContainer data={this.state.stars} />}/>
```

Comment:
For lines 91-94, `this.state` should always  be `this.state.photos`

We only need one property in state.